v-next/hardhat/test/hre/index.ts

```diff
@@ -0,0 +1,33 @@
+import assert from "node:assert";
```

Comment:
We should use `node:assert/strict` and remove "strict" from the assertions (`deepEqual` instead of `deepStrictEqual`).

Reply:
There's now a new eslint rule for this. So merging this will make the linter break in `v-next`. Do you mind changing this before merging, @zoeyTM ?